### PR TITLE
Support batched updates using PATCH operation.

### PR DESCRIPTION
**Pull request by jessemyers**
Triggered by a `--batch-size` of more than one.
Uses OPTIONS to validate that this operation is supported.

### CODE REVIEW

File: microcosm_resourcesync/endpoints/http_endpoint.py [hunk 1]
```diff
@@ -90,18 +96,73 @@ def write(self, resources, **kwargs):
...
-        NB: batch support via the BulkUpdate convention (PATCH) not implemented yet
         """
+        try:
+            # attempt to write resources in a batch
+            if len(resource_batch) > 1:
+                self.write_resource_batch(resource_batch, **kwargs)
+                return
+        except BatchingNotSupported:
+            # fall through
+            pass
```
Comment (netama) [inline microcosm_resourcesync/endpoints/http_endpoint.py]: why not raise?

**Comment by netama** [comment]
LGTM, one question

File: microcosm_resourcesync/endpoints/http_endpoint.py (hunk 1 above)
Comment (jessemyers) [inline microcosm_resourcesync/endpoints/http_endpoint.py]: We want to fallback to PUT one at a time.

Comment (netama) [inline microcosm_resourcesync/endpoints/http_endpoint.py]: :+1: I misread the flow

**Pull request closed without merging**